docs: Add guides example about #423 (#424)
LordOfPolls/Dis-Snek: docs/src/Guides/20 Scales.md

bot.start("token")
```

> Nowadays, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.

Reviewer: what?

Reviewer: I dont think this means what you think it means

bot.start("token")
```

> Use also can pass any **keyword** arguments into the `setup` function. Here is a basic "Scale switching" example:

Reviewer: Grammatical Error. Do you mean `you can also pass` or `users can also pass`?

bot.start("token")
```

> Additionally, for the cool bit of Scales, reloading. Scales allow you to edit your code, and reload it, without restarting the bot.

Reviewer 2: "Additionally" here is... confusing. I think "Now" or something similar would be better.

Nitpick though.

To do this, simply run `bot.regrow_scale("Filename_here")` and your new code will be used. Bare in mind any tasks your scale
is doing will be abruptly stopped.

> Finally, you can also pass any **keyword** arguments into module's `setup` and `teardown` functions, it will also send them into your `Scale.shed` method.

Reviewer 2: Am I the only one who got a bit confused by this? I'm *genuinely* asking by the way, it took me a bit to understand but I think that's just me.

Reviewer: It's grammatically fragmented. It could be rewritten with clearer wording

Reviewer 3: I understand it, but it's definitely not clean

Author: How to make it clearer?

Reviewer: My suggestion: 
https://github.com/Discord-Snake-Pit/Dis-Snek/pull/424#discussion_r845777544

> Finally, you can also pass any **keyword** arguments into module's `setup` and `teardown` functions, it will also send them into your `Scale.shed` method.

Reviewer: ```suggestion
Any keyword-arguments you pass to the `setup` or `teardown` methods, will also be passed to the `Scale.shed` method. 
```

Here is a basic "Scale switching" example:

```python
from dis_snek import Scale

class SomeScale(Scale):
    def __init__(self, bot, some_arg: int = 0):
        ...

class AnotherScale(Scale):

>     def __init__(self, _, another_arg: float = 0.0):

Reviewer 3: Why are you using `_` instead of bot here?

Reviewer: according to pep this would actually be correct, seeing as you dont use the argument, its purely there because `__new__` requires it to be passed. Though it could be argued as the below code is literally an ellipsis placeholder the true argument name should be used

Reviewer 3: For the sake of a readable example, we should definitely use the name.
